### PR TITLE
fixes #14616 - correct handling of EOF error

### PR DESCRIPTION
**Pull request by thomasmckay**
To test, import config-files on rhel7

### CODE REVIEW

File: lib/hammer_cli_import/configfile.rb [hunk 1]
```diff
@@ -136,39 +136,52 @@ def clean_module(name)
...
         # metadata.json file found at option_working_dir/<name>/metadata.json
         metadata_path = File.join(File.join(option_working_directory, name), 'metadata.json')
-        answer = @interview_answers['description'].gsub('#{module_name}', module_name)
-        sed_cmd = "sed -i '\/\"summary\":\/a \\ \\ \"description\": \"#{answer}\",' #{metadata_path}"
-        debug "About to issue #{sed_cmd}"
-        system sed_cmd
-        report_summary :wrote, :puppet_modules
+        if File.exist? metadata_path
+          answer = @interview_answers['description'].gsub('#{module_name}', module_name)
+          sed_cmd = "sed -i '\/\"summary\":\/a \\ \\ \"description\": \"#{answer}\",' #{metadata_path}"
```
Comment (beav) [inline lib/hammer_cli_import/configfile.rb]: can this be rewritten using `%q()` syntax? just to remove backslashes for readability

Comment (thomasmckay) [inline lib/hammer_cli_import/configfile.rb]: Readability does not describe this code. :) At this point I'm not in favor of any chance of adding accidental errors.

File: lib/hammer_cli_import/configfile.rb [hunk 2]
```diff
@@ -372,7 +387,7 @@ def build_and_upload
                                [data['org_id'].to_i, data['channel_id'].to_i])
 
           # Find the built-module .tar.gz
-          built_module_path = File.join(File.join(module_dir, 'pkg'),
+          built_module_path = File.join('pkg',
```
Comment (beav) [inline lib/hammer_cli_import/configfile.rb]: for my understanding, the `chdir` on line 165 makes it ok to remove the `module_dir` here?

Comment (thomasmckay) [inline lib/hammer_cli_import/configfile.rb]: Correct

**Comment by beav** [comment]
I don't have a full setup to test this but it looks good to me from a code perspective, given that we don't want to make large changes (see https://github.com/Katello/hammer-cli-import/pull/27#discussion_r70644724).

**Pull request merged**